share/trick/makefiles/trickify.mk

```diff
@@ -119,14 +119,20 @@ TRICK_EXT_LIB_DIRS :=
 .PHONY: all
 all: $(TRICKIFY_OBJECT_NAME) $(TRICKIFY_PYTHON_DIR)
 
-$(TRICKIFY_OBJECT_NAME): $(SWIG_OBJECTS) $(IO_OBJECTS) | $(dir $(TRICKIFY_OBJECT_NAME))
+$(BUILD_DIR)S_library_swig_objects: $(BUILD_DIR)S_library_swig
```

Comment:
Although this works in solving my specific use-case, these `echo ... | tr ...` lines could themselves result in `Argument list too long` if the library / build path was long enough. Without being able to use `$(file ...)` I don't know of a better way to do this, if you have suggestions let me know.